odxtools/diagdatadictionaryspec.py

```diff
@@ -86,18 +119,25 @@ def _build_id_lookup(self):
 
     def _resolve_references(self, parent_dl, id_lookup: dict):
         for dop in chain(
-                self.dtc_dops,
-                self.data_object_props,
-                self.tables,
+            self.dtc_dops,
+            self.data_object_props,
+            self.tables,
```

Comment:
it would be good if you made a separate PR for such formatting fixes in the future. Also they seem to be inconsistent, cf. e.g., line 95...

Reply:
Apologies. It habitual that I format the code every time. I'll avoid that in the future PRs.

Reply:
thanks. (it's not terribly bad with this PR but it detracts from the relevant changes...)